autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -343,6 +343,11 @@ def fit(train_data,
                 Keyword arguments to pass to all models. See the `AG_args_fit` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
                 Identical to specifying `AG_args_fit` parameter for all models in `hyperparameters`.
                 If a key in `AG_args_fit` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
+            excluded_model_types : list, default = []
+                Banned subset of model types to avoid training during `fit()`, even if present in `hyperparameters`.
+                Reference `hyperparameters` documentation for the list of model types.
```

Comment:
May be more user-friendly to list the model-types here since it probably doesn't take much more space than this sentence.

Reply:
We may also want to consider sorting the kwargs documentation based on what proportion of users we expect will utilize them?  E.g. kwargs like `excluded_model_types`,`id_columns` are probably far more commonly-used than `feature_generator_type`,`trainer_type`

Reply:
+1, improved language and re-ordered arguments